org.garudalinux.firedragon.appdata.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.garudalinux.firedragon</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0 AND LicenseRef-proprietary=https://raw.githubusercontent.com/Floorp-Projects/Floorp-private-components/main/LICENSE</project_license>
+  <name>FireDragon</name>
+  <summary><![CDATA[Floorp fork build using custom branding & settings]]></summary>
+  <developer_name>FireDragon Team</developer_name>
+  <icon type="stock">org.garudalinux.firedragon</icon>
+  <description>
+    <p><![CDATA[FireDragon is a browser based on the excellent Floorp browser. It was customized to have dr460nized-fitting aestetics as well as many opiniated settings by default. As this browser was originally a Librewolf fork, we are trying to integrate its best patches and tweaks in the new base. The most important features in addition of Floorps own ones can be found below.]]></p>
+    <ul>
+      <li><![CDATA[Searx & Whoogle search engines added, with the possibility to run locally]]></li>
```

Comment:
this is INVALID markup.

Reply:
Could you please elaborate? The docs clearly show `p`, `ul` and `li` tags and the linter did pass as well.

Reply:
![image](https://github.com/flathub/flathub/assets/114441/36ff7a3c-1953-4136-8602-5646c33bed3a)

Reply:
Fixed in 03091ea28a0fa2eb68fa1eec3d1ead5ac04bef6b.